### PR TITLE
updating the ccv_bootcamp workflow and requirements files for summer …

### PR DESCRIPTION
**Pull request by JRWallace**
…2024

### CODE REVIEW

File: .github/workflows/ccv_bootcamp-summer-tag.yml [hunk 1]
```diff
@@ -4,17 +4,17 @@ on: [workflow_dispatch]
...
+  SQLITE: false
+  PYTHON_VERSION: "3.9.17"
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-        #https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/
+      - uses: actions/checkout@v3.5.3
```
Comment (gtdang) [inline .github/workflows/ccv_bootcamp-summer-tag.yml:14]: The latest checkout version is 4. Any reason to keep it locked to this version? 

https://github.com/actions/checkout

**Comment by gtdang** [comment]
@digicosmos86 @prasadbandarkar @mirestrepo 
Could we Geta review on this so we can start testing out the hub?  Thank you!

File: .github/workflows/ccv_bootcamp.yml [hunk 2]
```diff
@@ -12,16 +12,17 @@ on:
...
-  PYTHON_VERSION: 3.7
+  SQLITE: false
+  PYTHON_VERSION: "3.9.17"
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3.5.3
```
Comment (digicosmos86) [inline .github/workflows/ccv_bootcamp.yml:22]: Maybe also v4 here?

**Review by digicosmos86 — APPROVED** [review]
Looks good to me!

**Pull request merged**